src/JointAxis_TEST.cc

```diff
@@ -264,6 +281,38 @@ TEST(DOMJointAxis, ToElement)
   ASSERT_TRUE(errors.empty());
   EXPECT_DOUBLE_EQ(1.5, dissipation);
 
+  // Check //axis/mimic
+  sdf::ElementPtr mimicElem = elem->GetElement("mimic", errors);
```

Comment:
Use `FindElement` since `GetElement` would create the `mimic` element if it doesn't exist. It would also be good to assert that the pointer is not nullptr.

Reply:
https://github.com/gazebosim/sdformat/pull/1166/commits/7fd569e420820d0668af2b7e3f342d3f46a9f5e2